Tools/Create-Azure-Sentinel-Solution/V3/CCP_README.md

```diff
@@ -0,0 +1,329 @@
+      }
+    },
+    "request": {
+      "apiEndpoint": "<Enter your api url>",
+      "httpMethod": "GET",
+      "queryParameters": {
+        "max_results": 100
+      },
+      "queryWindowInMin": 5,
+      "queryTimeFormat": "yyyy-MM-ddTHH:mm:ss.000000+00:00",
+      "startTimeAttributeName": "since",
+      "endTimeAttributeName": "until",
+      "rateLimitQps": 10,
+      "retryCount": 3,
+      "timeoutInSeconds": 60,
+      "headers": {
+        "Accept": "application/json",
+        "User-Agent": "Scuba"
+      }
+    },
+    "response": {
+      "eventsJsonPaths": ["$"],
+      "format": "json"
+    },
+    "paging": {
+      "type": "LinkHeader"
+    }
+  }
+}]
+```
+
+  **c. Data Collection Rules(DCR):**
+  - The "type" property value should be "Microsoft.Insights/dataCollectionRules".
+  - DCR, "name" property should be very short and without space. Total length of "name" field is 65 which on deployment adds  "Microsoft-Sentinel-{DCR-file-name-property-value}-{workspaceName}-{random-unique-value}". Here, "Microsoft-Sentinel" suffix is attached to the name property from DCR file and then attached with workspace name on which it is deployed along with a random number. When total length of this value exceeds then DCR will not get created and will fail in deployment.
+
+    ![Alt text](dcrFields.png)
+
+  - Once deployment is done it is recommended to verify if DCR is created in Azure portal, global search by searching for "Data Collection Rules" and then in "Data Collection Rules" search with the name that you specified in the source file on DCR "name" property. If search result shows a record in deployed resource group. If you don't see a result for the "name" property value specified in DCR file then its possibly a problem in DCR source file which failed in DCR creation. The first possible thing to verify if the length of DCR name exceed 65 characters.
+  - To verify length of DCR name, when we open data connector and click on "Connect" button, make sure to open browser "Developer tools" or right click on browser and do "Ctrl+F12" which will open up "Developer tools" and navigate to "Network" tab as shown below:
+  - Property "workspaceResourceId" is optional and is not required to be specified. If not specified then it replaces this property with value "[resourceId('microsoft.OperationalInsights/Workspaces', parameters('workspace'))]" in mainTemplate as a variables.
+  - Placeholder "{{location}}" will get replaced and will be specified in "parameters" section on "mainTemplate.json" file.
+  - Keep rest all properties of the file as is.
+  - Below is the sample file for DCR:
+
+```json
+[{
+  "name": "<Name of your DCR File Without Space>",
+  "apiVersion": "2021-09-01-preview",
+  "type": "Microsoft.Insights/dataCollectionRules",
+  "location": "{{location}}",
+  "properties": {
+    "dataCollectionEndpointId": "{{dataCollectionEndpointId}}",
+    "streamDeclarations": {
+      "Custom-<streamNameFromPollerFile>": {
```

Comment:
What is this "Custom-"  in the stream name? Is it not user specific?

Reply:
How about the Standard Tables? Can you provide clear info about this?

Reply:
updated